File: Assignments/Assignment_14/p4.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                int count=i;
                for(int j=1;j<=row;j++){

                        if(count==(row+1)){

                                count=1;
                        }
                        printf("%d\t",count);
                        count++;

                }printf("\n\n");
        }
}

File: JAVA/8.Nested_For_1/p4.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		int count=1;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=0;i<row;i++){
		
			for(int j=0;j<row;j++){
			
				System.out.print(count +"\t");
				count++;
			}System.out.println();
		}
	}
}
